msteams-platform/toolkit/tools-prerequisites.md

```diff
@@ -44,6 +44,13 @@ Ensure that you have the following accounts before you start building your Teams
 |[Azure account](#azure-account)|Back-end resources on Azure.| JavaScript, TypeScript, SPFx, and C# or Blazor. |
 |[SharePoint collection site administrator account](#sharepoint-collection-site-administrator-account) |Deployment for hosting.| SPFx. |
 
+> [!NOTE]
+> If your Microsoft 365 account has multiple tenants, you could switch to either tenant to develop Teams app. But there're several known limitations that should be aware of:
+>
+> * Multi-tenant is not supported by TDP yet. You may not be able to see your Teams app provisioned in non-home tenants from Teams Dev portal.
```

Comment:
You can use Teams Toolkit to deploy Teams app to guest tenant of M365 account, while you may not be able to see your Teams app deployed to non-home tenants in Teams Developer Portal. This is a limitation of Teams Developer Portal now.